### ISSUE TITLE
SentenceLabelDataset issues?

### ISSUE THREAD

**Issue by PrestonMWF** [UKPLab/sentence-transformers#323]
I've been using offline triplets with TripletReader to fine-tune embeddings with modest success because I can't get SentenceLabelDataset to work. For example, the triplet reader correctly picks up sentences longer than the max sequence length but SentenceLabelDataset does not. I want to do online training with the new triplet loss functions for better results/efficiency but the results are always poorer than out of the box embeddings. Any thoughts here? I saw in another issue that it might not be working so thought I'd check.
